Source/Core/DolphinQt2/Settings/InterfacePane.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <QWidget>
+
+class QCheckBox;
+class QComboBox;
+class QVBoxLayout;
+
+class InterfacePane final : public QWidget
+{
+  Q_OBJECT
+public:
+  explicit InterfacePane(QWidget* parent = nullptr);
+  void LoadConfig();
+
+private slots:
+  void SaveConfig();
```

Comment:
This can be a private function. Also typically try to make event handling functions have an `On` in their name. i.e. `OnSaveConfig`